src/theme/createSaleorTheme/createSaleorTheme.ts

```diff
@@ -145,7 +141,9 @@ export const createTheme = (colors: SaleorThemeColors): SaleorTheme =>
             color: colors.font.default,
             fontWeight: 400,
           },
-          borderRadius: 4,
+          "@media(min-width: 600px)": {
```

Comment:
Shouldn't we reference our breakpoints?

E.g.
```tsx
import { createBreakpoints } from "@material-ui/core/styles"

const breakpoints = createBreakpoints({})
```

```suggestion
          [breakpoints.up("sm")]: {
```

Reply:
It's hardcoded, so referencing it to our custom breakpoints (because defining them makes them custom) gives a false conviction that this will change if breakpoint would change.